src/main/java/springboot/SpringBootIntroHwApplication.java

```diff
@@ -2,28 +2,31 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
-import springboot.config.AppConfig;
 import springboot.model.Book;
 import springboot.service.BookService;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
 @SpringBootApplication
```

Comment:
```suggestion
@SpringBootApplication
@RequiredArgsConstructor
```

Reply:
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.AnnotationConfigApplicationContext;

without these imports, the compiler doesn't accept mvn clean package. Is that correct?"

Reply:
I didn't comment imports, I suggested to add `@RequiredArgsConstructor`

Reply:
Imports look like remove, because you have implementation of this HW in master branch 

https://github.com/Nebn1x/Spring-Boot-intro-hw/blob/master/src/main/java/springboot/SpringBootIntroHwApplication.java